spec/serializers/yellow_ribbon_program_serializer_spec.rb

```diff
@@ -49,11 +65,43 @@
     expect(attributes['name_of_institution']).to eq(yellow_ribbon_program.name_of_institution)
   end
 
+  it 'online_all' do
+    expect(attributes['online_all']).to eq(yellow_ribbon_program.online_all)
+  end
+
+  it 'online_only' do
+    expect(attributes['online_only']).to eq(yellow_ribbon_program.online_only)
+  end
+
   it 'state' do
     expect(attributes['state']).to eq(yellow_ribbon_program.state)
   end
 
   it 'street_address' do
     expect(attributes['street_address']).to eq(yellow_ribbon_program.street_address)
   end
+
+  it 'student_veteran' do
+    expect(attributes['student_veteran']).to eq(yellow_ribbon_program.student_veteran)
+  end
+
+  it 'student_vet_grp_ipeds' do
+    expect(attributes['student_vet_grp_ipeds']).to eq(yellow_ribbon_program.student_vet_grp_ipeds)
+  end
+
+  it 'student_veteran_link' do
+    expect(attributes['student_veteran_link']).to eq(yellow_ribbon_program.student_veteran_link)
+  end
+
+  it 'ungeocodable' do
+    expect(attributes['ungeocodable']).to eq(yellow_ribbon_program.ungeocodable)
+  end
+
+  it 'year_of_yr_participation' do
+    expect(attributes['year_of_yr_participation']).to eq(yellow_ribbon_program.year_of_yr_participation)
+  end
+
+  it 'zip' do
+    expect(attributes['zip']).to eq(yellow_ribbon_program.zip)
```

Comment:
Our Team will need to have an emergency meeting with Darla on Monday before we'll be able to approve any changes.

Reply:
Can somebody answer my JSON schema question?

Reply:
> Can somebody answer my JSON schema question?

Sorry for the slow reply.
- The yellow_ribbon_program schema used by unit test code has been updated in the latest commit.
- I don't see any other schemas in this repo that need updating.
- The corresponding Swagger response in vets-api needs to be updated, I'll implement that next.

Reply:
We need to do some analysis on our side to determine the impact of this change to our front end.